conda/cli/activate.py

```diff
@@ -62,7 +81,7 @@ def main():
             print(os.environ['PATH'])
             raise
         paths = []
-        sys.stderr.write("discarding %s from PATH\n" % binpath)
+        sys.stderr.write("discarding %s from PATH\n" % pathlist_to_str(binpath))
```

Comment:
I would like to keep it so that things are functional with NO conda/miniconda/anaconda path on PATH.

My concept for this is based around shortcuts to activate the root environment, much the same as Visual Studio does things.

This means that activate.bat from a raw terminal with no modified PATH will not work.  I think that is OK - people making the choice to not alter PATH is a reasonably advanced decision.

Do you have an opinion on this?

All this said, we need to be careful to keep the root environment on PATH, if the user has it on PATH to begin with.